Makefile

```diff
@@ -1,2 +1,13 @@
 lint:
 	golangci-lint run
+
+clean:
+    # '-f' to ignore when 'candebot' is not found
```

Comment:
this line must be indented by a tab, not spaces

Reply:
Good call